Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:14.04
+
+ADD ./WcaOnRails /app/WcaOnRails
+
+ENV agi="apt-get install --yes --no-install-recommends"
+
+# From https://www.brightbox.com/docs/ruby/ubuntu/
+RUN $agi software-properties-common
+RUN apt-add-repository --yes ppa:brightbox/ruby-ng
+
+RUN apt-get update
```

Comment:
these both make sense, I think once we have the stuff mostly working we should do this, for now it would be slightly annoying because it will redo the whole apt-get install when we add a single new dependency